scRNA-seq-advanced/01-read_filter_normalize_scRNA.Rmd

```diff
@@ -235,14 +259,14 @@ In this case, we will use our `mito_genes` vector to define a subset called `mit
 The `mito` name is important in that it is the name that will be expected by a later function.
 (We could define more subsets, but for now this one will do.)
 
-```{r per cell QC}
+```{r per cell QC, live=TRUE}
 filtered_sce <- scuttle::addPerCellQC(filtered_sce,
                                       subsets = list(mito = mito_genes))
 ```
 
 Now we can look at the colData to see what was added:
 
-```{r view coldata stats}
+```{r view colData stats}
 head(colData(filtered_sce))
 ```
 
```

Comment:
There is a typo on line 273 that I can't comment on `plots` should be `plot` towards the end of the line.